src/App.js

```diff
@@ -1,21 +1,25 @@
 import './App.css';
 import React from 'react';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import store from './redux/store';
 import Navbar from './components/Navbar';
 import Profile from './pages/Profile';
 import Rocket from './pages/Rocket';
 import Mission from './pages/Mission';
 
 function App() {
   return (
-    <Router>
-      <Navbar />
-      <Routes>
-        <Route exact path="/" element={<Rocket />} />
-        <Route exact path="/missions" element={<Mission />} />
-        <Route exact path="/profile" element={<Profile />} />
-      </Routes>
-    </Router>
+    <Provider store={store}>
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Rocket />} />
+          <Route exact path="/missions" element={<Mission />} />
+          <Route exact path="/profile" element={<Profile />} />
+        </Routes>
+      </Router>
+    </Provider>
```

Comment:
Kindly implement the `<Provider />` and the `<Router />` component in `index.js`. This would make the `App.js` look more cleaner and easier to read. This would also help you perform the changes in the next comment.